cloud/integrations.md

```diff
@@ -0,0 +1,136 @@
+product: cloud
+keywords: [integrations, DataDog, AWS CloudWatch]
+---
+
+import AttachExporter from 'versionContent/_partials/_cloud-integrations-attach-exporter.mdx';
+import ExporterRegionNote from 'versionContent/_partials/_cloud-integrations-exporter-region.mdx';
+
+# Integrate Timescale Cloud services with third-party monitoring tools
+
+You can export your service telemetry to a third-party monitoring tool, such as
+[DataDog][datadog] or [AWS CloudWatch][cloudwatch]. Exported metrics include
+CPU usage, RAM usage, and storage.
+
+## Export telemetry data to DataDog
+
+Export telemetry data to DataDog by:
+
+1.  Creating a data exporter
+1.  Attaching your database service to the exporter
+
+<procedure>
+
+### Creating a data exporter for DataDog
+
+<ExporterRegionNote />
+
+1.  In the Timescale Cloud console, navigate to `Integrations`.
+1.  Click `Create exporter`.
+1.  Under `Choose a provder`, choose `Datadog`.
+1.  Choose an AWS region for your exporter to live within Timescale Cloud. The
+    exporter is only available to database services in the same AWS region.
+1.  Name your exporter. This name appears in the Cloud console, so choose a
+    descriptive name.
+1.  Add a DataDog API key. If you don't have an API key yet, click `Click here
+    to generate a DataDog API key`.
+1.  Under Site, choose your DataDog region. You can choose a region to meet any
+    regulatory requirements or application needs you might have.
+1.  Click `Create endpoint`.
+
+<!-- FIXME: insert screenshot here -->
+
+</procedure>
+
+<AttachExporter />
+
+## Export telemetry data to AWS CloudWatch
+
+Export telemetry data to AWS CloudWatch by:
+
+1.  Creating a data exporter
+1.  Attaching your database service to the exporter
+
+<procedure>
+
+### Creating a data exporter for AWS CloudWatch
+
+<ExporterRegionNote />
+
+1.  In the Timescale Cloud console, navigate to `Integrations`.
+1.  Click `Create exporter`.
+1.  Under `Choose a provder`, choose `AWS CloudWatch`.
+1.  Choose an AWS region for your exporter to live within Timescale Cloud. The
+    exporter is only available to database services in the same AWS region.
+1.  Name your exporter. This name appears in the Cloud console, so choose a
+    descriptive name.
+1.  Define names for your CloudWatch log group, log stream, and namespace. If
+    you're uncertain, use the default values. For more information on naming log
+    groups and logs streams, see [the AWS CloudWatch
+    docs][cloudwatch-log-naming].
+1.  Enter your AWS credentials. To get your AWS keys, see [Getting access keys
+    for AWS CloudWatch][aws-access-keys].
+1.  Select an AWS Region for your CloudWatch instance.
+1.  **OPTIONAL** Define an Identity and Access Management (IAM) role to upload
+    segments to a different account.
+
+<!-- FIXME: insert screenshot here -->
+
+</procedure>
+
+<procedure>
+
+### Getting access keys for AWS CloudWatch
+
+1.  Navigate to your AWS Console.  
```

Comment:
there are a couple more steps to this

1. go to AWS console
2. open "IAM" console
3. in navigation pane select "users" --> "add user"
4. give user a user name
5. ... steps you already described

also, this is the link to the AWS docs in case we want to link there instead
https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console